### PR TITLE
[tests-only] prepare required resources rather than using skeleton dirs in API webdav properties and upload test features

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
With this PR:
- skeleton file/folders are used more wisely on the following features suite
  - apiWebdavProperties1
  - apiWebdavProperties2
  - apiWebdavUpload1
  - apiWebdavUpload2
  - apiWebdavUploadTUS

## Related Issue

- Part of https://github.com/owncloud/QA/issues/658

## How Has This Been Tested?

- :robot

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by saw-jan — COMMENTED** [review]
LGTM

File: tests/acceptance/features/apiWebdavProperties1/getQuota.feature [hunk 1]
```diff
@@ -21,7 +21,7 @@ Feature: get quota
   Scenario Outline: Retrieving folder quota when quota is set
     Given using <dav_version> DAV path
     When the administrator sets the quota of user "Alice" to "10 MB" using the provisioning API
-    Then as user "Alice" folder "/" should contain a property "d:quota-available-bytes" with value "10485406"
+    Then as user "Alice" folder "/" should contain a property "d:quota-available-bytes" with value "10485597"
```
Comment (Talank) [inline tests/acceptance/features/apiWebdavProperties1/getQuota.feature:24]: is this value supposed to change if no skeleton dir is used? 🤔

Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavProperties1/getQuota.feature:24]: quota available differs/increases while using user which is initialized without skeleton files

**Review by Talank — COMMENTED** [review]
LGTM

**Review by individual-it — APPROVED** [review]

**Pull request merged**